src/ngx-gallery-scroll-overview/zoomPosition.ts

```diff
@@ -0,0 +1,4 @@
+export class ZoomPosition {
```

Comment:
This is more an interface than a class: `export interface ZoomPosition`. And please change the filename to `zoom-position.interface.ts`